api/services/Search.js

```diff
@@ -38,9 +39,14 @@ module.exports = {
       }
 
       if (opts.is_public) {
-        qb.whereIn('is_public', opts.is_public)
+        qb.where('is_public', opts.is_public)
       }
 
+      filterAndSortCommunities({
+        search: opts.term,
+        sortBy: opts.sort,
```

Comment:
this seems like it is duplicating above filters, just a little worried it might cause issues, sorting one way and then another, or searching by something extra. maybe just take the search and sortBy out?

Reply:
in fact why isnt this code just using filterAndSortByCommunities for all of it?